applications/graphQL/type.py

```
@@ -0,0 +1,53 @@
+            "nsfw",
+            "added_on",
+            "added_by",
+            "last_modified",
+            "status",
+        )
+        filterset_class = InsultFilter
+
+    # Add computed fields if needed
+    is_active = Boolean(description="Indicates if the insult is currently active")
+
+    def resolve_is_active(self, info) -> bool:
```

Ruff (0.12.2)

Missing type annotation for function argument `info`

(ANN001)